tests/component_tests/deep_sleep/test_deep_sleep_esp8266_wifi.yaml

```
@@ -0,0 +1,24 @@
+---
+esphome:
+  name: test_esp8266_wifi_deep_sleep
+
+esp8266:
+  board: esp12e
+
+wifi:
+  ssid: "test"
+  password: "test"
```

Wifi passwords need to be at least 8 characters
```suggestion
  password: "testtest"
```